packages/docs/src/examples/color-variables/ColorBorder.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <div style="border: 1px solid var(--va-success);">Use --va-success</div>
```

Comment:
Maybe you can make something more interesting here.

![image](https://user-images.githubusercontent.com/23530004/125070007-3015a680-e0c0-11eb-872d-69f6f62dc29c.png)

```
<template>
  <div style="border: 1px solid var(--va-danger); background-color: var(--va-warning);">

    <i class="mx-2" style="color: var(--va-danger);">!</i>

    <span style="color: var(--va-white);">This is custom alert that uses CSS variables</span>
  </div>
</template>
```

Reply:
Also, you can pass props to an example component, that will be used instead of "This is custom alert...". 

We don't have this option for now, but you are free to add it. You can add it to ExampleOptions if you want.

![image](https://user-images.githubusercontent.com/23530004/125070928-4708c880-e0c1-11eb-8159-2b696afddbf4.png)